### PR TITLE
Release notes for Coopr 0.9.9

### PR DESCRIPTION
**Pull request by wolf31o2**

### CODE REVIEW

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 1]
```diff
@@ -25,14 +25,135 @@
...
-
 Release Highlights
 ------------------
-  * 1
-  * 2
+  * Docker support, ability to deploy Docker services and manage Docker containers
+  * Provider plugin for Digital Ocean added
+  * New UI, rewritten to be more modular and extensible
+  * CLI tool, allows interaction via simple commands
+  * Split Server APIs into internal and external, served on different ports
```
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst]: Splits the Server APIs into internal and external APIs, served on different ports

(at first, I thought it was a new type of "Split Server"...)

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 2]
```diff
@@ -25,14 +25,135 @@
...
+  * Node usage metrics, provides detailed metrics by tenant, user, or template
```
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:35]: by either tenant, user, or template

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 3]
```diff
@@ -25,14 +25,135 @@
...
-  * 1 (Issues: #xx #xx)
+
+New Features
+^^^^^^^^^^^^
+
+  * `COOPR-195 <https://issues.cask.co/browse/COOPR-195>`_ - Entity versioning
+  * `COOPR-199 <https://issues.cask.co/browse/COOPR-199>`_ - Ability to suspend jobs
+  * `COOPR-418 <https://issues.cask.co/browse/COOPR-418>`_ - Add more detailed usage statistics
+  * `COOPR-493 <https://issues.cask.co/browse/COOPR-493>`_ - Add a command line interface
+  * `COOPR-496 <https://issues.cask.co/browse/COOPR-496>`_ - Add some way to expose links for a cluster
```
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:54]: Add a method to expose links for a cluster

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 4]
```diff
@@ -25,14 +25,135 @@
...
+  * `COOPR-497 <https://issues.cask.co/browse/COOPR-497>`_ - Split out sensitive config setting into a separate config file
+  * `COOPR-513 <https://issues.cask.co/browse/COOPR-513>`_ - Add labels to admin entities
+  * `COOPR-531 <https://issues.cask.co/browse/COOPR-531>`_ - Command Line Shell for Coopr similar to CDAP CLI
+  * `COOPR-542 <https://issues.cask.co/browse/COOPR-542>`_ - Add ability to configure UI to use TLS
+  * `COOPR-563 <https://issues.cask.co/browse/COOPR-563>`_ - Support for DigitalOcean provider
+  * `COOPR-567 <https://issues.cask.co/browse/COOPR-567>`_ - Split server APIs to separate ports.
```
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst]: Splits the server APIs to two separate ports (and drop period)

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 5]
```diff
@@ -25,14 +25,135 @@
...
+
+Improvements
+^^^^^^^^^^^^
+
+  * `COOPR-64 <https://issues.cask.co/browse/COOPR-64>`_ - Confirmations should mention item being acted upon
+  * `COOPR-300 <https://issues.cask.co/browse/COOPR-300>`_ - Move lease times away from milliseconds
+  * `COOPR-338 <https://issues.cask.co/browse/COOPR-338>`_ - Improve disk handling for AWS
+  * `COOPR-367 <https://issues.cask.co/browse/COOPR-367>`_ - Standalone should copy loom-site.xml.example into server/conf
+  * `COOPR-500 <https://issues.cask.co/browse/COOPR-500>`_ - Include organization in pom
+  * `COOPR-537 <https://issues.cask.co/browse/COOPR-537>`_ - Add documentation to Cluster Pause feature
```
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:80]: Add documentation of Cluster Pause feature (? not sure, maybe "of"?)

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 6]
```diff
@@ -25,14 +25,135 @@
...
+  * `COOPR-545 <https://issues.cask.co/browse/COOPR-545>`_ - Better handling of memory settings for Coopr Server
+  * `COOPR-556 <https://issues.cask.co/browse/COOPR-556>`_ - remove CORS
+  * `COOPR-600 <https://issues.cask.co/browse/COOPR-600>`_ - CLI: doesn't show current user or tenant
+  * `COOPR-601 <https://issues.cask.co/browse/COOPR-601>`_ - CLI: alias ? to TAB
```
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:84]: alias "?" to TAB

**Comment by awholegunch** [comment]
A few comments, otherwise LGTM.

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst (hunk 2 above)
Comment (wolf31o2) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:35]: It's not `either`, at all. You can use any of them, none of them, or combinations of them.

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst (hunk 3 above)
Comment (wolf31o2) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:54]: That's the JIRA's title... these are unedited. Should I still edit it? My gut says "leave it"

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst (hunk 2 above)
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:35]: maybe then, "tenant, user, template, or a combination of these" ?

Comment (wolf31o2) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:35]: or none...

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst (hunk 3 above)
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:54]: Leave it.

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst (hunk 2 above)
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst:35]: then perhaps:

```
Node usage metrics, provides detailed metrics, optionally by tenant, user, template, or any combination of these
```

**Pull request merged**